### PR TITLE
use highest res possible output

### PR DESCRIPTION
**Pull request by AWare**
So, the output for this tool was quite lo-res, so what I've done is:
- removed the scale parameter and multiplied it into the constants it was used in 
- calculated an output size and scale factor from the image and used these everywhere
- added a css scale transform to the canvas to restore its size
- bonus: changed the port to not conflict with metro bundler

### CODE REVIEW

File: src/canvas.js [hunk 1]
```diff
@@ -66,149 +84,195 @@ class CanvasCard {
...
-        font: Config.headline.font,
-        fontSize: Config.headline.fontSize[headlineSize]
+      const { width, height, scale } = this._getCanvasDimensions({
+        deviceWidth,
+        deviceHeight,
+        imageHeight: image.height,
+        imageWidth: image.width
       });
+      const canvas = this._getNewCanvas({ width, height });
+      console.log(scale);
```
Comment (akash1810) [inline src/canvas.js]: is this needed?

**Comment by AWare** [comment]
Moved port and prettier to #14 and #13

File: src/canvas.js (hunk 1 above)
Comment (AWare) [reply src/canvas.js]: Removed

**Comment by AWare** [comment]
This is needed as the existing sized assets were getting destroyed by the image resizer. Maybe the hidpi will help, but we're already producing big enough assets in this pr so I'm not sure if that would be a useful change.

**Pull request merged**